instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+import './PostContainer.css'
+import Comment from '../CommentSection/CommentComponent'
+import moment from 'moment'
+
+let time
+
+function Post(props) {
+  return (
+    <div className='post-container'>
+      <header className='top-bar'>
+        <img src={props.post.thumbnailUrl} alt='user image' className='profileimg'></img> 
+        <h3>{props.post.username}</h3>       
+      </header>
+
+      <img src={props.post.imageUrl} alt='posted image'></img>
+
+      <div className='pngImgs'>
+        <img src='./219-heart.png' alt='like'></img>
+        <img src='./111-bubble2.png' alt='comment'></img>
+      </div>
+
+      <div className='comment'>
+        { <h3>{props.post.likes} Likes</h3>}
+        {/* <p><strong>{props.post.comments[0].username}</strong> {props.post.comments[0].text}</p> */}
```

Comment:
i like this comment here, its gets really important to layout your code and know what you want where in bigger projects, 
the difference between a decent developer and a great one is how well they utilize their comments in code.